### ISSUE TITLE
How to remove some warnings?

### ISSUE THREAD

**Issue by amty** [bbatsov/guru-mode#13]
It would be nice to have a way to remove some of the affected keys from a guru list. For example, I like the idea to teach myself to not use Home/End/PgUp/PgDn, because this keys are missing from or in weird locations on laptop keyboards, but I see no reason to unlearn arrow keys, because I use them in any case for moving between windows by S-<arrow> keybindings. But if I tried to ignore some of guru's warnings, it would lead to ignoring all of them.

I've removed offending keys for now by editing guru sources, but it would be nice to do it 'properly' from init.el.

**Comment by bbatsov** [comment]
You can simply modify `guru-affected-bindings-list`.

**Comment by amty** [comment]
You mean do something like `(setq guru-affected-bindings-list (seq-filter ... guru-affected-bindings-list))` in init.el? It doesn't work.

**Comment by bbatsov** [comment]
Yep, but you you need to do it before enabling `guru-mode`, as this var is used by its `guru-init` function.

**Comment by amty** [comment]
I tried at first, but have found that guru-affected-bindings-list was unbound.

**Comment by bbatsov** [comment]
Can you paste here how you structured the configuration exactly?
